setup.py

```diff
@@ -7,4 +7,20 @@
     install_requires=[
         'requests~=2.5',
     ],
+    extras_require={
+        "dev": [
+            "cdislogging==1.0.0",
+            "markupsafe~=1.1.1",         # newer version not compatible
+            "pre-commit~=1.21.0",
+            "pytest~=6.2.0",
+            "sqlalchemy~=1.3.0",
+            "sqlalchemy-utils>=0.32,<0.36.4",
+            "indexd @ git+https://github.com/NCI-GDC/indexd.git@2.9.0-rc.2#egg=indexd",
+        ],
+        "test_utils2": [
```

Comment:
If we plan on getting rid of the original index_test_utils package, since this doesn't seem to conflict with anything else would it make sense to just name it `test_utils`?

Reply:
Sounds good